src/components/DetailContainer.js

```diff
@@ -25,7 +25,7 @@ class DetailContainer extends Component {
   render () {
     const { children } = this.props
     return (
-      <div className={'container-fluid ' + sharedStyle['move-left-detail']}>
+      <div className={'container-fluid ' + sharedStyle['move-left-detail'] + ' ' + sharedStyle['detail-z-index']}>
```

Comment:
Can be left for a follow-up, but string composition with back-ticks is better in this case